packages/manager/src/features/VPC/VPCCreate/VPCCreate.tsx

```diff
@@ -101,13 +84,23 @@ const VPCCreate = () => {
       const response = await createVPC(createVPCPayload);
       history.push(`/vpc/${response.id}`);
     } catch (errors) {
-      const apiSubnetErrors = errors.filter(
+      const indivSubnetErrors = convertVpcSubnetApiErrors(
+        errors,
+        values.subnets.length,
+        setFieldError
+      );
+      // must combine errors and subnet data to avoid indexing weirdness when deleting a subnet
+      const subnetsAndErrors = combineErrorsAndSubnets(
+        values.subnets,
+        indivSubnetErrors
+      );
+      setFieldValue('subnets', subnetsAndErrors);
```

Comment:
todo Connie - if we go with this validation route, look into combining this into one method